install/update_91_92.php

```diff
@@ -1294,6 +1294,26 @@ function update91to92() {
       $migration->dropTable('glpi_networkequipmentfirmwares');
    }
 
+   //add editor in followupps
+   if (!fieldExists('glpi_ticketfollowups', 'editors_id')) {
```

Comment:
should be in upper case FieldExists, but the underlying question is more : with this function is not called fieldExists ?